### DISCUSSION TITLE
For some reason cronjob is not called

### DISCUSSION THREAD

**Discussion by arstruct-onishi** [supabase/supabase#17389] [Questions]
Hi,

I tried to below code, but cache_user_insight function is not called...
Why and how should I do next ?

CREATE OR REPLACE FUNCTION cache_user_insight()
RETURNS VOID LANGUAGE plpgsql AS $$
DECLARE
	response text;
BEGIN
	response := net.http_post(
		url:='https://xxxxxx.functions.supabase.co/cache-user-insight',
		headers:='{"Content-Type": "application/json", "Authorization": "Bearer xxxxxx"}'::jsonb,
		body:=concat('{"time": "', now(), '"}')::jsonb
	);
END;
$$;

SELECT cron.unschedule( 'cron-cache-user-insight' );
SELECT cron.schedule( 'cron-cache-user-insight', '0 * * * *', 'SELECT cache_user_insight();' );

Thanks Regards,

**Comment by TheOtherBrian1** [comment]
Why is your Select statement in quotes? Below is an example of a cron job calling a PG_NET function. Just substitute the `net.http_get... ` with your function's name. 

```sql
SELECT cron.schedule(
	'cron-job-name',
	'* * * * *', -- Executes every minute (cron syntax)
	$$
	    -- SQL query
	    SELECT net.http_get(
		-- URL of Edge function
		url:='https://<reference id>.functions.Supabase.co/example',
		headers:='{
		    "Content-Type": "application/json",
		    "Authorization": "Bearer <TOKEN>"
		}'::JSONB
	    ) as request_id;
	$$
);
```
